### PR TITLE
Add Bytom Sidechain network (BMC)

### PR DESCRIPTION
**Pull request by MartinBTM**
add BMC network information

Fixes # .

Changes proposed in this pull request:

- add BMC network information

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -580,6 +580,20 @@
...
+  },  
+  "188": {
+    "key": "188",
+    "name": "Bytom Sidechain",
+    "shortName": "BMC",
+    "chainId": 188,
+    "network": "mainnet",
+    "multicall": "0x10cd4fa0D02eD0f33e5b569C371194F39898Ff3b",
+    "rpc": [
+      "https://mainnet.bmcchain.com/"
```
Comment (ChaituVR) [inline src/networks.json:592]: Looks like this is not a full archive node @MartinBTM 
It give me `missing trie node c5bd7f02c614571ed4ff4693e2ed3801391ad5eb9fb3d88d54be94fed8193633 (path` error if i try older block

File: src/networks.json [hunk 2]
```diff
@@ -580,6 +580,20 @@
...
+    ],
+    "explorer": "https://bmc.blockmeta.com/",
+    "start": 4500000,
```
Comment (ChaituVR) [inline src/networks.json:595]:
```suggestion
    "start": 7113,
```

File: src/networks.json (hunk 1 above)
Comment (ChaituVR) [reply src/networks.json:592]: Also something wrong with multicall, it give this error everytime
```
Transaction reverted without a reason string [
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/networks.json (hunk 1 above)
Comment (MartinBTM) [reply src/networks.json:592]: sure，Thanks， we will check it

Comment (MartinBTM) [reply src/networks.json:592]: we have changed to archive mode now, and deploy a new mutil call, please check the pr ,Thanks very much

File: src/networks.json [hunk 3]
```diff
@@ -580,6 +580,20 @@
...
+    "shortName": "BMC",
+    "chainId": 188,
+    "network": "mainnet",
+    "multicall": "0xDA09528B093246eC70139b657d3B7A3bd5F4C859",
+    "rpc": [
+      "https://mainnet.bmcchain.com/"
+    ],
+    "explorer": "https://bmc.blockmeta.com/",
+    "start": 4720651,
+    "logo": "bafkreiabhsxuq35pp4kmrbptdeypd6clhcy3ue7ydpppo6onoo4igcjqia"
```
Comment (ChaituVR) [inline src/networks.json:596]:
```suggestion
    "logo": "ipfs://bafkreiabhsxuq35pp4kmrbptdeypd6clhcy3ue7ydpppo6onoo4igcjqia"
```

File: src/networks.json [hunk 4]
```diff
@@ -580,6 +580,20 @@
...
     "start": 9834491,
     "logo": "ipfs://QmaGokGqgjknfa4xnXKnnwC5ZyXzUjQ7p6KEe4D8G5uFFE"
+  },  
+  "188": {
+    "key": "188",
+    "name": "Bytom Sidechain",
+    "shortName": "BMC",
+    "chainId": 188,
+    "network": "mainnet",
+    "multicall": "0xDA09528B093246eC70139b657d3B7A3bd5F4C859",
```
Comment (ChaituVR) [inline src/networks.json:590]: Is there a way to check contract code here https://bmc.blockmeta.com/address/0xDA09528B093246eC70139b657d3B7A3bd5F4C859

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/networks.json (hunk 4 above)
Comment (MartinBTM) [reply src/networks.json:590]: yes

Comment (ChaituVR) [reply src/networks.json:590]: @MartinBTM I mean verify and publish the contract code to the explorer

Comment (MartinBTM) [reply src/networks.json:590]: hi，we have push the code to explorer, please check

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**